### DISCUSSION TITLE
August 17th 2023 PowerShell Community Call

### DISCUSSION THREAD

**Discussion by StevenBucher98** [PowerShell/PowerShell#20004] [Community Call]
Agenda to come
- Bug Bash recap @SydneyhSmith (Thank you community for joining!)
- PSResourceGet Release @SydneyhSmith 
- Crescendo RC release @theJasonHelmick 
- DSC v3 @mgreenegit 
- Docs Update @sdwheeler 
- VSCode Extension preview release @andyleejordan 
- Azure PowerShell version update in-tool notification @dcaro 
- Conference recap @sdwheeler

**Comment by SteveL-MSFT** [comment]
Charlie Kindel to demo updates to ConsoleGuiTools module

**Reply by tig** [reply]
Confirmed!

**Reply by tig** [reply]
@SteveL-MSFT What time is this at on the 17th? I'm Mountain Time.

**Reply by ThomasNieto** [reply]
It would be 10:30 for mountain time.

**Comment by HeyItsGilbert** [comment]
Any chance we could toss the call notes into the youtube description? I have a job that shares to our internal Workplace group whenever a new video is posted and having the notes there would make it easy for me to include that in the post.

**Reply by StevenBucher98** [reply]
We usually post the notes in the RFC repo, https://github.com/PowerShell/PowerShell-RFC/tree/master/CommunityCall/notes,  but can also include them in the description of the videos 😄

**Reply by ThomasNieto** [reply]
@StevenBucher98 think we can also get the meeting chat added as well? It used to be there in the past but hasn't been for a while.

**Reply by SydneyhSmith** [reply]
@ThomasNieto in the past was the meeting chat included in the recording of did we also post a transcript of the chat?

**Reply by ThomasNieto** [reply]
@SydneyhSmith the transcript was posted as well, here is a link to the last [chat transcript](https://github.com/PowerShell/PowerShell-RFC/blob/master/CommunityCall/notes/20210318_ChatTranscript.txt) from 2021.

**Reply by SydneyhSmith** [reply]
Gotcha thanks, I am guessing we stopped doing this because its not a direct artifact of the recording when we switched from a teams live event to a meeting but we will try and recreate it

**Comment by StartAutomating** [comment]
If there's time, I'd love to cover the `$psStyle` related updates in [EZOut](https://github.com/StartAutomating/EZOut) or the cool formatting and features of [Posh](https://github.com/StartAutomating/Posh)

**Comment by AspenForester** [comment]
I'm hoping there might be a chance to remind the team responsible for the PS runspaces in Azure Automation that there really are some of us out in the world who really would like to see a 'supported' 7.x runspace.

**Reply by StevenBucher98** [reply]
I will follow up with the team

**Reply by AspenForester** [reply]
I appreciate it!

**Reply by n-bajaj** [reply]
Azure Automation supports PS7.2 runbooks in preview and we are targeting 9/30 for GA.

**Comment by tig** [comment]
I don't see the call connection info. Where do I find out how to connect?

**Reply by SydneyhSmith** [reply]
aka.ms/joinPSCall should work
